src/app/shared/result.actions.ts

```diff
@@ -0,0 +1,4 @@
+export class EditResultView {
+  static readonly type = '[Result] EditView';
+  constructor(public payload: string) {}
+}
```

Comment:
suggestion to use the empty line at the end of each file for avoiding conflicts when somebody adds code to the file